app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt

```diff
@@ -86,6 +88,7 @@ class BrokenSiteActivity : DuckDuckGoActivity() {
         val errorCodes = intent.getStringArrayExtra(ERROR_CODES).orEmpty()
         val httpErrorCodes = intent.getStringExtra(HTTP_ERROR_CODES).orEmpty()
         val isDesktopMode = intent.getBooleanExtra(IS_DESKTOP_MODE, false)
+        val reportFlow = intent.getSerializableExtra<ReportFlow>(REPORT_FLOW)!!
```

Comment:
We generally avoid `!!` in code (It's fine in tests), is there a way to handle the null case more gracefully?